tests/cft/test_r2_support_bundle.py

```diff
@@ -756,6 +756,105 @@ def test_31256(self):
         self.LOGGER.info("Successfully validated Motr rotating log files are as per "
                          "frequency configured for all pods")
 
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.lc
+    @pytest.mark.support_bundle
+    @pytest.mark.tags("TEST-34548")
+    def test_34548(self):
+        """
+        Validate support bundle “size_limit” filter
+        """
+        self.LOGGER.info("STARTED: Test to validate support bundle 'size_limit' filter")
+
+        self.LOGGER.info("Step 1: Generate support bundle")
+        pod_types_to_test = [constants.POD_NAME_PREFIX, constants.SERVER_POD_NAME_PREFIX]
+
+        for pod_type in pod_types_to_test:
+            pod_list = self.node_obj.get_all_pods(pod_prefix=pod_type)
+            machine_id = self.node_obj.get_machine_id_for_pod(pod_list[0])
```

Comment:
so instead of pod_list[0] can we randomly select pod from pod_list?

Reply:
yes, done